Dockerfile

```diff
@@ -1,22 +1,22 @@
 
-FROM ubuntu:latest
+FROM ubuntu:bionic
 
 ENV GCSFUSE_REPO gcsfuse-stretch
 
-RUN apt-get update && apt-get install --yes --no-install-recommends \
-    ca-certificates \
-    curl \
-    gnupg \
-    mydumper \
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN mkdir /backups && mkdir -p /mnt/backup-bucket && \
+    apt-get update && apt-get install --yes --no-install-recommends \
+    ca-certificates=20210119~18.04.2 \
+    curl=7.58.0-2ubuntu3.16 \
+    gnupg=2.2.4-1ubuntu1.4 \
+    mydumper=0.9.1-5 \
   && echo "deb http://packages.cloud.google.com/apt $GCSFUSE_REPO main" \
     | tee /etc/apt/sources.list.d/gcsfuse.list \
   && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
   && apt-get update \
-  && apt-get install --yes gcsfuse \
+  && apt-get install --yes gcsfuse=0.40.0 --no-install-recommends \
   && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* 
 
-RUN mkdir /backups && mkdir -p /mnt/backup-bucket
```

Comment:
does this result in some functional change removing this? I would have thought this might result in the `fuse` mounting needing this?

Reply:
Ah, I see. It moved up to line 7